content/resources/_index.nl.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "Kennisbank"
```

Comment:
YES! Kennisbank is such a great translation. I struggled trying to think of one on the old site :sweat_smile: